.github/workflows/ci.yml

```diff
@@ -21,3 +21,31 @@ jobs:
     with:
       license-check: true
       lint: true
+  bun:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Setup Bun
+        uses: oven-sh/setup-bun@v2
+
+      - name: Install dependencies
+        # The erraneous typod version `5.0.0-aplha` takes priority over .3 as 'p' > 'l'
+        run: |
+          bun i --ignore-scripts
+          bun i -D --ignore-scripts fastify@5.0.0-alpha.3
```

Comment:
This is not ideal

The typod version seems to be unpublished now — can't find it at https://www.npmjs.com/package/fastify?activeTab=versions

Can you try removing the line?

```suggestion
```

Reply:
I can confirm this is the case